osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/SecretManagementService.java

```diff
@@ -45,15 +45,13 @@
 /**
  * Service for storing, activating and retrieving device keys.
  * Also performs RSA encryption/decryption operations for SOAP messaging purposes.
- */
-public class SecretManagementService {
+ */ public class SecretManagementService {
```

Comment:
Sloppy formatting, the start of the class definition should not be on the same line as closing the javadoc.